.travis.yml

```diff
@@ -74,7 +68,8 @@ script:
     - mkdir build && cd build
 
     - if [ "$FLAVOR" == "cs" ]; then cmake -DBOND_SKIP_GBC_TESTS=TRUE -DBOND_SKIP_CORE_TESTS=TRUE -DBOND_ENABLE_COMM=FALSE -DBOND_ENABLE_GRPC=FALSE ..; fi
-    - if [ "$FLAVOR" == "cs" ]; then make --jobs 2 DESTDIR=$HOME install; fi
+    - if [ "$FLAVOR" == "cs" ]; then make gbc; fi
+    - if [ "$FLAVOR" == "cs" ]; then make DESTDIR=$HOME install; fi
```

Comment:
I'll spin it off as its own issue but I want to investigate a little bit more. Doing a sticky for now.